tools/template.xml.in

```diff
@@ -1,113 +1,120 @@
-  <uuid>d06d502a-904a-4b34-847d-debf1a3d76c7</uuid>
-  <memory unit='KiB'>2097152</memory>
-  <currentMemory unit='KiB'>2097152</currentMemory>
-  <vcpu placement='static'>4</vcpu>
+  <uuid>UUID</uuid>
+  <title>MACOSNAME</title>
+  <memory unit="KiB">4194304</memory>
+  <currentMemory unit="KiB">4194304</currentMemory>
+  <vcpu placement="static">4</vcpu>
   <os>
-    <type arch='x86_64' machine='MACHINE'>hvm</type>
-    <loader readonly='yes' type='pflash'>VMDIR/firmware/OVMF_CODE.fd</loader>
-    <nvram>VMDIR/firmware/OVMF_VARS-1024x768.fd</nvram>
-    <boot dev='hd'/>
+    <type arch="x86_64" machine="MACHINE">hvm</type>
+    <loader readonly="yes" type="pflash">QEMUHOME/firmware/OVMF_CODE.fd</loader>
+    <nvram>QEMUHOME/nvram/OVMF_VARS-1024x768.fd</nvram>
+    <boot dev="hd" />
   </os>
   <features>
-    <acpi/>
-    <apic/>
-    <vmport state='off'/>
+    <acpi />
+    <apic />
+    <vmport state="off" />
   </features>
   <cpu>
-    <topology sockets='1' cores='4' threads='1'/>
+    <topology sockets="1" cores="4" threads="1" />
   </cpu>
-  <clock offset='utc'>
-    <timer name='rtc' tickpolicy='catchup'/>
-    <timer name='pit' tickpolicy='delay'/>
-    <timer name='hpet' present='no'/>
+  <clock offset="utc">
+    <timer name="rtc" tickpolicy="catchup" />
+    <timer name="pit" tickpolicy="delay" />
+    <timer name="hpet" present="no" />
   </clock>
   <on_poweroff>destroy</on_poweroff>
   <on_reboot>restart</on_reboot>
   <on_crash>destroy</on_crash>
   <pm>
-    <suspend-to-mem enabled='no'/>
-    <suspend-to-disk enabled='no'/>
+    <suspend-to-mem enabled="no" />
+    <suspend-to-disk enabled="no" />
   </pm>
   <devices>
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
-    <disk type='file' device='disk'>
-      <driver name='qemu' type='qcow2'/>
-      <source file='VMDIR/ESP.qcow2'/>
-      <target dev='sda' bus='sata'/>
-      <address type='drive' controller='0' bus='0' target='0' unit='0'/>
+    <disk type="file" device="disk">
+      <driver name="qemu" type="qcow2" cache="writeback" io="threads" />
+      <source file="BOXESHOME/ESP.qcow2" />
+      <target dev="sda" bus="sata" />
+      <address type="drive" controller="0" bus="0" target="0" unit="0" />
     </disk>
-    <disk type='file' device='disk'>
-      <driver name='qemu' type='raw'/>
-      <source file='VMDIR/BaseSystem.img'/>
-      <target dev='sdb' bus='sata'/>
-      <address type='drive' controller='0' bus='0' target='0' unit='1'/>
+    <disk type="file" device="disk">
+      <driver name="qemu" type="qcow2" cache="writeback" io="threads" />
+      <source file="BOXESHOME/macOS.qcow2" />
+      <target dev="sdb" bus="sata" />
+      <address type="drive" controller="0" bus="0" target="0" unit="1" />
     </disk>
-    <controller type='usb' index='0' model='ich9-ehci1'>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x1d' function='0x7'/>
+    <disk type="file" device="disk">
+      <driver name="qemu" type="raw" />
```

Comment:
Raw disks are not supported by snapshots, so GNOME boxes  as well as CLI would fail to create a snapshot:

```
virsh snapshot-create-as --domain 'macOS-Simple-KVM' --name "macOS-fresh-install"
error: unsupported configuration: internal snapshot for disk sdc unsupported for storage type raw
```

Reply:
It used for the first installation, but it can be removed after install.